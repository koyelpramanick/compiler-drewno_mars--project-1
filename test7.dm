while (guy == false) {
    num++;
    butt = "this is a correct string"
} //this is a right paren
"bad escape oh no! \o" 67
today I don't feel like doing any work at all!
"let's fail to terminate this string
24Kmagic 600000000000000
"bad escape \q and unterminated...
//i hope that's all