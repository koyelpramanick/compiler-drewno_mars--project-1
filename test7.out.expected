WHILE [1,1]
LPAREN [1,7]
ID:guy [1,8]
EQUALS [1,12]
FALSE [1,15]
RPAREN [1,20]
LCURLY [1,22]
ID:num [2,5]
POSTINC [2,8]
SEMICOL [2,10]
ID:butt [3,5]
ASSIGN [3,10]
STRINGLITERAL:"this is a correct string" [3,12]
RCURLY [4,1]
INTLITERAL:67 [5,24]
EXIT [6,1]
ID:at [6,40]
ID:all [6,43]
NOT [6,46]
MAGIC [8,1]
INTLITERAL:0 [8,10]
EOF [10,20]
